### ISSUE TITLE
Export VPTW-DEV-SDP requests

### ISSUE THREAD

**Issue by colton-kognic** [annotell/public-docs#59]
The following requests in [VPTW-DEV-SDP](https://tools.app.kognic.com/projects/740) are ready to be exported now. You might have exported them already but I was not sure.
6053, 
5588 (already done I think),
5521,
5517,
5516,
5515,
5514,
5513

**Comment by colton-kognic** [comment]
Blocked by https://github.com/annotell/kognic-soleng/issues/522
